lib/bolt/bolt_option_parser.rb

```diff
@@ -6,6 +6,31 @@
 
 module Bolt
   class BoltOptionParser < OptionParser
+    INCLUDE_OPTS = { 'apply' => %w[nodes targets query rerun noop description execute user password private-key
```

Comment:
I think it'd be useful to bundle these into the categories they already have in the help text ('authentication', 'run_context', etc.) and then maybe have a 'common_opts' or something the bundles those together as well, since most of these share most of their options.

Reply:
agreed. I think this can be fine grained based on specific characteristics or more focussed on what we need right now. 

I think there is a large group of options that are used by all "targetable actions"
targetting options: nodes/target/query/rerun
transport options: user/password/private-key/transport/...
inventoryfile/save-rerun